quinn/examples/connection.rs

```diff
@@ -1,17 +1,6 @@
-//! This example intends to use the smallest amount of code to make a simple QUIC connection.
+//! Checkout the `README.md` for guidance.
```

Comment:
I meant this:

```suggestion
//! This example intends to use the smallest amount of code to make a simple QUIC connection.
//!
//! Checkout the `README.md` for guidance.
```

(And skip the link; same for the other instances here.)